solutions/CF Mixing Water.mdx

```diff
@@ -0,0 +1,138 @@
+---
+id: cf-1359C
+source: CF
+title: Mixing Water
+author: Jesse Choe
+---
+
+[Official Editorial](https://codeforces.com/blog/entry/78116)
```

Comment:
i think you can delete this, it's probably not necessary to link it in two places